index.html

```diff
@@ -1445,7 +1445,7 @@ <h5 style="margin:auto 4px">KSA</h5>
 
     function getStandings()
     {
-        const url = `${baseUrl}/competitions/2000/standings`
+        const url = `https://cors-anywhere.herokuapp.com/${baseUrl}/competitions/2000/standings`
```

Comment:
This demo of CORS Anywhere should only be used for development purposes

Reply:
same?!